Seminar5/Task3.py

```diff
@@ -0,0 +1,16 @@
+# Напишите однострочный генератор словаря, 
+# который принимает на вход три списка одинаковой длины: имена str, ставка int, премия str с 
+# указанием процентов вида “10.25%”. 
+# В результате получаем словарь с именем в качестве ключа и суммой 
+# премии в качестве значения. 
+# Сумма рассчитывается как ставка умноженная на процент премии
+
+names = ['Ivan', 'Serg', 'Petr']
+salarys = [1000, 2000, 3000]
+awards = ['10.25%', '10.00%', '10.5%']
+my_dict ={}
+my_gen = ({name: salary * float(award[:-1]) / 100} for name, salary, award in zip(names, salarys, awards))
```

Comment:
А зачем круглые скобки?
```suggestion
my_gen = {name: salary * float(award[:-1]) / 100} for name, salary, award in zip(names, salarys, awards)
```